lib/tpm/key_attestation.rb

```diff
@@ -15,7 +15,7 @@ class KeyAttestation
       begin
         pattern = File.expand_path(File.join(__dir__, "certificates", "*", "RootCA", "*.*"))
         Dir.glob(pattern).map do |filename|
-          File.open(filename) { |file| OpenSSL::X509::Certificate.new(file) }
+          File.binread(filename) { |file| OpenSSL::X509::Certificate.new(file) }
```

Comment:
so passing a block to `File.open` changes the return value to that of the block, but that's not the case with `File.read` or `File.binread`.

Which is why the CI is failing.

To fix the issue we can write this as
```ruby
file = File.binread(filename)
OpenSSL::X509::Certificate.new(file)
```

Reply:
Nice! Thanks 🙂 

I'll take a look

Reply:
created https://github.com/cedarcode/tpm-key_attestation/pull/21 to fix the pipeline.